src/libos/src/events/waiter_queue.rs

```diff
@@ -13,15 +13,15 @@ use crate::prelude::*;
 /// of `Waker`s.
 pub struct WaiterQueue {
     count: AtomicUsize,
-    wakers: SgxMutex<VecDeque<Waker>>,
+    wakers: SgxMutex<LinkedList<Waker>>,
```

Comment:
Would you please kindly elaborate more on why we'd prefer `LinkedList` to `VecDeque` for `wakers`? On which case will it better perform?